lib/tests.rb

```diff
@@ -115,6 +115,11 @@ def handle_finished_tests(tests)
     print_tests_stats
   end
 
+  def prevent_clients_not_ready
```

Comment:
Same here: reset_clients_to_ready_state or clients_reset_to_ready_state or if this is client class already just reset_to_ready_state

Reply:
On it...